apps/etterlatte-utbetaling/src/test/kotlin/utbetaling/TestHelper.kt

```diff
@@ -194,4 +201,48 @@ fun utbetalingshendelse(
     utbetalingId: UUID = UUID.randomUUID(),
     tidspunkt: Tidspunkt = Tidspunkt.now(),
     status: UtbetalingStatus = UtbetalingStatus.MOTTATT
-) = Utbetalingshendelse(id, utbetalingId, tidspunkt, status)
\ No newline at end of file
+) = Utbetalingshendelse(id, utbetalingId, tidspunkt, status)
+
+fun mockKonsistensavstemming(
```

Comment:
Hvorfor ikke brucke `mockk` til mocking av objekt?

Reply:
Kunne gjort det, men denne brukes flere steder, hvor noen er avhengig av fullt kopulerte objekter for å kunne kjøre tester.

Reply:
Syns selv det er bedre å bruke riktige objekter når man har mulighet istedet for mockk-objekter, sån att man faktiskt har äkta data med funksjonalitet som objektet har. 😊 👍